src/install/local/setup.md

```diff
@@ -0,0 +1,43 @@
+# Setup Environment
+----
+1. Create a cluster where the Engine can be installed. For our example, we'll use kind.
```

Comment:
feels like we could use some headers here to jump around the page easier

Reply:
As apposed to a numbered list? 
This page does feel funky.